internal/k8s/k8s.go

```diff
@@ -245,9 +245,12 @@ func New(cfg *Config) (*Client, error) {
 	}
 
 	http.Handle("/metrics", promhttp.Handler())
-	go func() {
-		http.ListenAndServe(fmt.Sprintf("%s:%d", cfg.MetricsHost, cfg.MetricsPort), nil)
-	}()
+	go func(l log.Logger) {
+		err := http.ListenAndServe(fmt.Sprintf("%s:%d", cfg.MetricsHost, cfg.MetricsPort), nil)
+		if err != nil {
+			l.Log("Cannot listen and server on %s:%d", cfg.MetricsHost, cfg.MetricsPort) //nolint:errcheck
```

Comment:
Could you use the existing logging style? We can discuss changing it but for now let's stay consistent.

Also - `s/server/serve/`.

Reply:
Done. I hadn't notice the logging style. In fact, what I had was a bug because the arguments are supposed to be K/V pairs.